### PR TITLE
TeamsShiftsPolicy: Initial Release

### PR DESCRIPTION
**Pull request by NikCharlebois**
#### Pull Request (PR) description
* New TeamsShiftsPolicy Resource

#### This Pull Request (PR) fixes the following issues
* N/A

### CODE REVIEW

File: Modules/Microsoft365DSC/DSCResources/MSFT_TeamsShiftsPolicy/MSFT_TeamsShiftsPolicy.psm1 [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+    )
+
+    try
+    {
+        New-M365DSCConnection -Workload 'MicrosoftTeams' `
+            -InboundParameters $PSBoundParameters | Out-Null
+    }
+    catch
+    {
+        Write-Verbose -Message $_
```
Comment (ykuijs) [inline Modules/Microsoft365DSC/DSCResources/MSFT_TeamsShiftsPolicy/MSFT_TeamsShiftsPolicy.psm1:68]: Why do you only write a verbose here? If the catch is triggered, doesn't that mean creating the connection failed? And in that case, shouldn't the rest of the code be aborted?

File: Modules/Microsoft365DSC/DSCResources/MSFT_TeamsShiftsPolicy/MSFT_TeamsShiftsPolicy.psm1 [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
```
Comment (ykuijs) [inline Modules/Microsoft365DSC/DSCResources/MSFT_TeamsShiftsPolicy/MSFT_TeamsShiftsPolicy.psm1:189]: Same comment as before

**Review by ykuijs — CHANGES_REQUESTED** [review]
Question needs clarification

File: Modules/Microsoft365DSC/DSCResources/MSFT_TeamsShiftsPolicy/MSFT_TeamsShiftsPolicy.psm1 (hunk 2 above)
Comment (NikCharlebois) [reply Modules/Microsoft365DSC/DSCResources/MSFT_TeamsShiftsPolicy/MSFT_TeamsShiftsPolicy.psm1:189]: Yes, it should. This Teams resource was entirely generated via the Dynamic Resource Generator. I will regenerate with your comments included.

File: Modules/Microsoft365DSC/DSCResources/MSFT_TeamsShiftsPolicy/MSFT_TeamsShiftsPolicy.psm1 (hunk 1 above)
Comment (NikCharlebois) [reply Modules/Microsoft365DSC/DSCResources/MSFT_TeamsShiftsPolicy/MSFT_TeamsShiftsPolicy.psm1:68]: To be aligned with the other resources, we will remove the try/catch. The error will be trapped inside the New-M365DSCCOnnection function instead

**Review by ykuijs — APPROVED** [review]
LGTM

**Pull request merged**